recipe/meta.yaml

```diff
@@ -35,7 +35,8 @@ requirements:
     # boost max_pin set in pin_run_as_build in conda-forge-pinning
```

Comment:
In the host requirements, we should change cpptango >= 9.4.1.  9.4.0 won't work properly.

Reply:
Of course. I forgot :-(